src/dotnet/APIView/APIViewWeb/LeanModels/ChangeHistory.cs

```diff
@@ -12,11 +12,54 @@ public enum AICommentChangeAction
         Modified
     }
 
-    public class AICommentChangeHistoryModel
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum ReviewChangeAction
+    {
+        Created = 0,
+        Closed,
+        ReOpened,
+        Approved,
+        ApprovalReverted,
```

Comment:
Is this to track approval for first preview release? APIs are approved at revision level. right?

Reply:
Yes, this is approval for first release. But we no longer need to make that distinction. Reviews can be approved, and revisions can be approved. Unless You are asking to know why Review approval will be reverted which I guess should never happen.